emcc.py

```diff
@@ -2348,7 +2358,7 @@ def repl(m):
       # Generate the fetch.js worker script for multithreaded emscripten_fetch() support if targeting pthreads.
       if shared.Settings.FETCH and shared.Settings.USE_PTHREADS:
```

Comment:
This should be `if shared.Settings.USE_FETCH_WORKER`. No point creating module.fetch.js if it's not going to be used.

Reply:
Good idea, thanks, done.